server/src/main/java/org/opensearch/telemetry/tracing/channels/TraceableTcpTransportChannel.java

```diff
@@ -47,6 +50,7 @@ public TraceableTransportChannel(TransportChannel delegate, Span span, Tracer tr
      * @param delegate delegate
      * @param span     span
      * @param tracer tracer
+     * @param tcpChannel tcpChannel
      * @return transport channel
      */
     public static TransportChannel create(TransportChannel delegate, final Span span, final Tracer tracer, final TcpChannel tcpChannel) {
```

Comment:
```suggestion
    public static TransportChannel create(TcpTransportChannel delegate, final Span span, final Tracer tracer) {
```

And you could access channel using `delegate.getChannel()`